awesomeConfig.cmake

```diff
@@ -71,6 +71,13 @@ if (NOT LUA_FOUND)
         "You might want to hint it using the LUA_DIR environment variable, "
         "or set the LUA_INCLUDE_DIR / LUA_LIBRARY CMake variables.")
 endif()
+
+set(LUA_FULL_VERSION "${LUA_VERSION_MAJOR}.${LUA_VERSION_MINOR}.${LUA_VERSION_PATCH}")
+if(LUA_FULL_VERSION VERSION_EQUAL 5.4.0 OR LUA_FULL_VERSION VERSION_GREATER 5.4.0 )
```

Comment:
Note that there's `VERSION_GREATER_EQUAL`, which requires CMake 3.7 though, and we only require 3.0 currently.  So good to have it more verbose/expanded here.

Reply:
Yup, I thought it'd be better not to assume cmake version, especially that it doesn't make the code much longer